index.html

```diff
@@ -272,26 +273,29 @@ <h2>Resources</h2>
           <div class="resource">
             <h2>Legal</h2>
             <h3>Citizens Advice Bureau</h3>
-            <h4>The Relationship property – the basics section is useful to read though they don’t
-              mention trusts at all. Some CAB’s offer free half hour sessions with a lawyer.</h4>
+            <h4>Some CABs offer free half hour sessions with a lawyer.</h4>
               <a class="btn btn-primary" a href="http://www.cab.org.nz/Pages/home.aspx">Visit CAB</a>
             </div>
             <!-- <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4"> -->
             <div class="resource">
               <h3>Legal Aid</h3>
               <h4>Visit the Department of Justice website for information on your eligibility for legal aid support.</h4>
-              <a class="btn btn-primary" a href="https://www.justice.govt.nz/courts/going-to-court/legal-aid/">Visit website</a>
+              <a class="btn btn-primary" a href="https://www.justice.govt.nz/courts/going-to-court/legal-aid">Visit website</a>

```

Comment:
Heya, Remove the "a" before the href. For all the hrefs. 

Also the colours on the buttons display's differently. Not sure why.

Reply:
It should be a <div class="btn btn-primary><a></a></div>
Thats probs why the colours are different

Reply:
Ah thanks Janie I was scratching my head why the links were different colours.
But Anna-Marie - I don't get why to remove the <a> tag? Then it won't link??

Reply:
Hmmmm what have I done wrong here??
<div class="btn btn-primary" a href="https://www.familyservices.govt.nz/directory/">Visit website</a></div>
it is not linking to the href??

Reply:
Sorry here's the code example:
```<div class="btn btn-primary" a href="https://www.familyservices.govt.nz/directory/">Visit website</a></div>```

Reply:
<div class="btn btn-primary><a href=""></a></div>
See how the a is in a separate tag